libs/eslint-plugin-vx/src/configs/react.ts

```diff
@@ -31,6 +31,9 @@ export = {
     ],
     'react/jsx-filename-extension': ['error', { extensions: allExtensions }],
     'react/jsx-fragments': ['error', 'element'],
+    'react/jsx-no-bind': 'off',
+    'react/jsx-no-constructed-context-values': 'off',
+    'react/jsx-no-useless-fragment': ['error', { allowExpressions: true }],
```

Comment:
Allows e.g. `<React.Fragment>{foo}</React.Fragment>`.

Reply:
Does this mean we'll no longer get warnings when we have an unnecessary `React.Fragment`?

Reply:
It's an auto-fixable rule in the case where the fragment has exactly one element child; that's what fixed most of the violations in this commit. Enabling `allowExpressions: true` does mean that, in the event the `<React.Fragment>` is not needed when used with an expression instead of an element, the rule will allow it when it arguably shouldn't. I haven't seen any cases of this in vxsuite, though. We _do_ have some cases of using `<React.Fragment>` with expressions, which is why I enabled this exception.

Reply:
Ah gotcha, so that's for computing dynamic fragments? I see how that could be useful